### DISCUSSION TITLE
How to make and use fine-tuning model with spark nlp?

### DISCUSSION THREAD

**Discussion by leeivan** [JohnSnowLabs/spark-nlp#8446] [Q&A] [answered]
To fine-tuning specifically  data in gpt-2 model, this process could be done with huggingface transformer, then the model be used with spark nlp. Whether could this whole process be done with spark nlp totally?

**Comment by maziyarpanahi** [comment, marked as answer]
No, there are no fine tuning features within Java APIs for TensorFlow. Those features are only available for Python version of TensorFlow for now.
